python/salpytools/salpylib.py

```diff
@@ -5,7 +5,6 @@
 import logging
```

Comment:
LSST license header? https://developer.lsst.io/python/style.html#each-python-file-must-contain-the-standard-license-preamble